src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -654,17 +656,47 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("2.7");
 		}
+
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerStopUsingItemEvent")) {
+			Skript.registerEvent("Stop Using Item", SimpleEvent.class, PlayerStopUsingItemEvent.class,
+							"[player] (stop|end) (using item|item use)")
+					.description("Called when a player stops using an item. For example, when the player releases the " +
+							"interact button when holding a bow, an edible item, or a spyglass.",
+							"Note that event-number/event-timespan will return the number of ticks the item was held for.")
+					.requiredPlugins("Paper 1.18.2+")
+					.examples(
+						"on player stop using item:",
+							"\tbroadcast \"%player% held %event-item% for %event-timespan%.\"")
+					.since("INSERT VERSION");
+		}
+
+		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerReadyArrowEvent")) {
+			Skript.registerEvent("Ready Arrow", SimpleEvent.class, PlayerReadyArrowEvent.class, "[player] (ready|pick|draw) arrow")
```

Comment:
I don't know how I feel about `player pick arrow`, to me it makes me think that this event is for picking arrows from the ground. Maybe remove `pick` from the options?

Reply:
do you think "choose" is a good option?

Reply:
Actually I didn't notice this while reviewing.. the pick part is indeed not a good choice, I would suggest the following
- `choose` sounds good
- `loading` sounds good
- `[player] ((ready|choose|draw|loading) arrow|arrow (choose|draw|loading))`

Reply:
I'll probably use `load` to match tenses, but those sound good. 
(and re: missing things, turns out everyone missed that `entities` wasn't even being used in CondIsHandRaised and therefore the toString would always have null for the expression lol)

Reply:
Oh dear 😁 that's why I don't like reviewing big PRs

Reply:
yes this one got a bit out of hand